### ISSUE TITLE
Not working with ACF v5.4.5?

### ISSUE THREAD

**Issue by davidcie** [Hube2/acf-input-counter#6]
Not too familiar with either the plugin or ACF internals so apologies if I'm missing something obvious. I can see in dev tools that acf-input-counter.js is being loaded, my text field does have a limit of 160 characters (published the group twice after installing the plugin just to make sure) but I still cannot see counter appear.

Field definition:

![field_definition](https://cloud.githubusercontent.com/assets/1963866/18833880/344376bc-83f3-11e6-9d6c-195cedd9e6ab.jpg)

Field in use:

![field](https://cloud.githubusercontent.com/assets/1963866/18833875/3070f37a-83f3-11e6-8cdb-bca4058bafa5.jpg)

Let me know how I can help nail down the problem.

**Comment by Hube2** [comment]
I'm going to need to look at what the difference might be in ACF's html markup now. The appearance of the counter is dependent on that. Thanks for letting me know that it's not working.

**Comment by Hube2** [comment]
No, this is my fault for accepting a pull request without completely examining it. The person added a requirement to add a specific class to fields where you want the counter to display, but did not update the documentation to indicate it.

When I built this this plugin is just supposed to work without needing any additional work and that's the way I plan to keep it. I will remove the test for a class so it's back as I intended it. My opinion but things like this just need to work automatically, everywhere.

I will think about adding some type of filter so that people that want to do a little extra work. I may also add a filter so that people can alter the text that is displayed as well.
